micro_dl/cli/torch_inference_script.py

```diff
@@ -0,0 +1,101 @@
+import os
+import yaml
+import datetime
+import torch
+import micro_dl.inference.image_inference as image_inf
+import micro_dl.torch_unet.utils.inference as torch_inference_utils
+import micro_dl.utils.train_utils as train_utils
+
+import argparse
+
+def read_config(config_path):
```

Comment:
There's already a read config function in microDL so please use that: https://github.com/mehta-lab/microDL/blob/master/micro_dl/utils/aux_utils.py#L47